packages/vite-node/src/client.ts

```diff
@@ -539,8 +539,12 @@ export class ViteNodeRunner {
 
     this.options.moduleExecutionInfo?.set(options.filename, { startOffset: codeDefinition.length })
 
+    const start = performance.now()
+
     const fn = vm.runInThisContext(code, options)
     await fn(...Object.values(context))
+
+    this.options.moduleExecutionInfo?.set(options.filename, { startOffset: codeDefinition.length, duration: performance.now() - start })
```

Comment:
Awkward setting it twice like this, but reusing `moduleExecutionInfo` is convenient